compressor/conf.py

```diff
@@ -70,6 +70,12 @@ class CompressorConf(AppConf):
     OFFLINE_MANIFEST = 'manifest.json'
     # The Context to be used when TemplateFilter is used
     TEMPLATE_FILTER_CONTEXT = {}
+    # URL-less mode: you do offline compression & decompression regardless of settings.STATIC_URL
+    # and settings.COMPRESS_URL so your assets become re-locatable for any URLs.
+    # Affects manifest file contents and manifest key determination, but not the assets contents.
+    OFFLINE_URLLESS = False
```

Comment:
@diox just to make sure i get this right, you propose to remove the URLLESS setting altogether and just always do the replacement with no option to turn it off? as in, if we had seen earlier that this approach is cool, most of @kottenator's work wouldn't have been necessary ;) ? (i think we should do it this way, just am a bit annoyed that i/we didn't think of it earlier)

@kottenator can you think of any downsides?